core/ecschema-rpc/common/src/ECSchemaRpcLocater.ts

```diff
@@ -0,0 +1,37 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { ISchemaLocater, Schema, SchemaContext, SchemaKey, SchemaMatchType, SchemaProps } from "@bentley/ecschema-metadata";
+import { IModelRpcProps } from "@bentley/imodeljs-common";
+import { ECSchemaRpcInterface } from "./ECSchemaRpcInterface";
+
+/**
+ * Defines a schema locater that retrieves schemas using an RPC interface.
+ * @alpha
+ */
+export class ECSchemaRpcLocater implements ISchemaLocater {
+    public token: IModelRpcProps;
```

Comment:
Any reason for `token` to be mutable?

Reply:
You're right, it isn't necessary for it to be mutable. I'll change it to be `readonly`